examples/user_interaction/client/src/index.js

```diff
@@ -48,6 +48,8 @@ window.ocAgent = 'http://localhost:55678';
 // For the purpose of this example, send trace header to all hosts.
 window.ocTraceHeaderHostRegex = /.*/;
 window.ocSampleRate = 1.0; // Sample at 100% for test only. Default is 1/10000.
+// Sample user interactions at 100% for test only. Default is 1/10000.
+window.ocUserInteractionSampleRate = 1.0; 
```

Comment:
What would you think about just consolidating this with the `ocSampleRate` parameter such that a single sampling decision is made for whether to trace the initial load and all on-page interactions?